src/main/java/io/fabric8/forge/generator/keycloak/KeycloakEndpoint.java

```diff
@@ -26,15 +27,15 @@
     // http://sso.prod-preview.openshift.io/auth/realms/fabric8/account
     // authorization: Bearer <ACCESS_TOKEN>
     // http://sso.prod-preview.openshift.io/auth/realms/fabric8/account/identity
-    GET_OPENSHIFT_TOKEN("OpenShift", "/auth/realms/fabric8/broker/openshift-v3/token"),
-    GET_GITHUB_TOKEN("GitHub", "/auth/realms/fabric8/broker/github/token");
+    GET_OPENSHIFT_TOKEN("OpenShift", URLUtils.pathJoin(Configuration.keycloakSaaSURL(), "/auth/realms/fabric8/broker/openshift-v3/token")),
```

Comment:
I have a question: 
Previous we only had `"/auth/realms/fabric8/broker/github/token"`
and now we have `URLUtils.pathJoin(Configuration.keycloakSaaSURL(), "/auth/realms/fabric8/broker/openshift-v3/token"))` .

How did it work before with only the path? :)

Reply:
`GET_OPENSHIFT_TOKEN()` calls the enumeration constructor `KeycloakEndpoint(String name, String path)` where we had `this.endpoint = URLUtils.pathJoin(Configuration.keycloakSaaSURL(), path);`
Now we are removing it from constructor. See lines 37-38.
It's a Java thing ;)

Reply:
Oh right :D my mind was seeing this change in reverse - thanks for pointing out.
```
-        this.endpoint = URLUtils.pathJoin(Configuration.keycloakSaaSURL(), path);
+        this.endpoint = url;
```